phpcs.xml.dist

```diff
@@ -21,6 +21,8 @@
         <exclude name="SlevomatCodingStandard.PHP.RequireExplicitAssertion.RequiredExplicitAssertion" />
         <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingAnyTypeHint" />
         <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint" />
+        <exclude name="SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingTraversableTypeHintSpecification" />
+        <exclude name="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingAnyTypeHint" />
```

Comment:
I'd keep those. If there's any issue with them, we should address it in the CS lib

Reply:
The issue is that it does not respect inheritance, without using the lovely `@inheritDoc`.

Reply:
So we can keep inheritdoc for now, there's one or two. Better than omitting types everywhere else.

Reply:
We are not omitting types anywhere though. PHPStan has still got our back, PHP_CodeSniffer is just flawed in that regard and more trouble than its worth.

Reply:
hm, that's true. Is level 3 enough to check that? I'll do some tests eventually.

Reply:
When I remove `@param Promise[]|mixed[] $promisesOrValues Promises or values.` from `PromiseAdapter::all()` it is not detected unfortunatelly

https://github.com/webonyx/graphql-php/blob/c886d2bf0679f6dc010209c2e64961776c74bd27/src/Executor/Promise/PromiseAdapter.php#L85

Reply:
It comes in at level 6. We'll get there eventually 😉

Reply:
Eventually, but until then we're not covered by checks. Still disagree on disabling these until there's alternative in place.

Reply:
https://github.com/doctrine/coding-standard/pull/258